packages/sonarwhal/src/lib/utils/resource-loader.ts

```diff
@@ -261,6 +262,18 @@ export const loadResource = (name: string, type: ResourceType, configurations: A
             resource = res;
         }
 
+        if (resource && (source === currentProcessDir)) {
```

Comment:
we are already checking if resource exists or not.

Move this inside the if above.